### PR TITLE
[Demo] Memory error demo

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jnothman** [comment]
Thanks for opening all those issues.. you've requested review. What are we aiming for here?

**Comment by hmaarrfk** [comment]
just recreating how to produce the unknown error while I remember how.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,17 +40,44 @@ requirements:
     - scipy
     - joblib >=0.11
 
+
+# Some tests take alot of memory, and seem to cause segfaults when memory runs out
+{% set test_cpus = CPU_COUNT %}
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
{% set test_cpus = "32" %}
```

**Comment by hmaarrfk** [comment]
This PR is in reference to https://github.com/scikit-learn/scikit-learn/issues/15827

**Comment by jjerphan** [comment]
Hi @hmaarrfk, should this PR be closed since https://github.com/scikit-learn/scikit-learn/issues/15827 is resolved?

**Comment by hmaarrfk** [comment]
sure.

**Pull request closed without merging**